File: assets/www/Liquor/Models/Credential.js


function Credential() {

	var accountId;
	var credentialType;
	var token;
	var logged;
	
	this.getAccountId = function() {
		return accountId;
	}
	
	this.setAccountId = function(val) {
		accountId = val;
	}
	
	this.getCredentialType = function(val) {
		return credentialType;
	}
	
	this.setCredentialType = function(val) {
		credentialType = val;
	}
	
	this.getToken = function() {
		return token;
	}
	
	this.setToken = function(val) {
		token = val;
	}
	
	this.isLogged = function() {
		return logged;
	}
	
	this.setLogged = function(val) {
		logged = val;
	}

}

FunctionUtils.extend(Database, Credential);

File: connect-sample/iOS/Siminov/Connect/Sample/Model/Credential.m
//
//  Credential.m
//  connect-sample
//
//  Created by Geetika on 29/05/15.
//  Copyright (c) 2015 Siminov. All rights reserved.
//

#import "Credential.h"

@implementation Credential

- (NSString *)getAccountId {
    return accountId;
}

- (void)setAccountId:(NSString *) accountid {
    accountId = accountid;
}

- (NSString *)getToken {
    return token;
}

- (void)setToken:(NSString *)tok {
    token = tok;
}

- (BOOL)getActive {
    return active;
}

- (BOOL)isActive {
    return active;
}

- (void)setActive:(BOOL)act {
    active = act;
}

@end
